packages/abstractions/src/apiClientProxifier.ts

```diff
@@ -216,7 +217,7 @@ export interface RequestMetadata {
 	errorMappings?: ErrorMappings;
 	adapterMethodName?: SendMethods;
 	responseBodyFactory?: ParsableFactory<Parsable> | PrimitiveTypesForDeserialization;
-	requestBodySerializer?: ModelSerializerFunction<Parsable> | PrimitiveTypesForDeserialization;
+	requestBodySerializer?: ModelSerializerFunction<Parsable> | ModelSerializerFunction<MultipartBody> | PrimitiveTypesForDeserialization;
```

Comment:
since [multipartBody implements IParsable](https://github.com/microsoft/kiota-typescript/blob/1edca86f2e3e9007cfe6f132e28503850d42d4f3/packages/abstractions/src/multipartBody.ts#L14) what does this solve?

Reply:
+1 on the comment by @baywet 

There was an inconsistency with the type information earlier releases but that was resolved with the PR at https://github.com/microsoft/kiota-typescript/pull/1399 as the function signature was missing the null information.

Reply:
The build error was as a result of an older version